src/App.tsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Movie {
+  title: string;
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
+
+export function filterMovies(movies: Movie[], query: string): Movie[] {
+  const visibleMovies = movies.filter(movie => {
+    const { title, description } = movie;
+
+    const checkString = (string: string) => string
+      .toLowerCase()
+      .includes(query.toLowerCase().trim());
```

Comment:
I think it would be better to store this function in the upper scope, so you don't re-create it on each iteration